docs/usage/user-stories/maintaining-aur-packages-with-renovate.md

```diff
@@ -0,0 +1,172 @@
+People have created and used tools, like [`urlwatch`](https://github.com/thp/urlwatch) and [`aurpublish`](https://github.com/eli-schwartz/aurpublish), to let them know when upstream releases are cut and automate some parts of the process.
+I know I do.
+But I wanted to automate the entire process.
+I think [Renovate](https://github.com/renovatebot/renovate/) can help here.
+
+## Updating versions with Renovate
+
+Renovate is an automated dependency update tool.
+You might have seen it opening pull requests on GitHub and making updates for npm or other package managers, but it’s a lot more powerful than just that.
+
+Renovate has a couple of concepts that I need to explain first: [datasources](../modules/datasource/index.md) and [managers](../modules/manager/index.md).
+Datasources define where to look for new versions of a dependency.
+Renovate comes with over 50 different datasources, but the one that is important for AUR packages is the [`git-tags` datasource](https://docs.renovatebot.com/modules/datasource/#git-tags-datasource).
+Managers are the Renovate concept for package managers.
+There isn’t an AUR or `PKGBUILD` manager, but there is a [regex manager](https://docs.renovatebot.com/modules/manager/regex/) that I can use.
+
+I can create a `renovate.json` configuration with the following regex manager configuration:
+
+```json
+{
+  "regexManagers": [
+    {
+      "fileMatch": ["(^|/)PKGBUILD$"],
+      "matchStrings": [
+        "pkgver=(?<currentValue>.*) # renovate: datasource=(?<datasource>.*) depName=(?<depName>.*)"
+      ],
+      "extractVersionTemplate": "^v?(?<version>.*)$"
+    }
+  ]
+}
+```
+
+Breaking that down:
+
+- The `fileMatch` setting tells Renovate to look for any `PKGBUILD` files in a repository
+- The `matchStrings` is the regex format to extract the version, datasource, and dependency name from the `PKGBUILD`
+- The `extractVersionTemplate` is to handle a “v” in front of any version number that is sometimes added to Git tags
+
+And here’s an extract from the PKGBUILD for the [bicep-bin](https://aur.archlinux.org/packages/bicep-bin) AUR package that I maintain:
+
+```bash
+pkgver=0.15.31 # renovate: datasource=github-tags depName=Azure/bicep
+```
+
+Here I’m configuring Renovate to use the [`github-tags`](https://docs.renovatebot.com/modules/datasource/github-tags/) datasource and to look in the [Azure/bicep GitHub repository](https://github.com/Azure/bicep) for new versions.
+That means it’ll look in the [list of tags in that repository](https://github.com/Azure/bicep/tags) for any new versions.
+If Renovate finds any new versions, it’ll automatically update the `PKGBUILD` and open a pull request with the updated version.
+
+So I’ve automated the `PKGBUILD` update, but that’s only half of the work.
+The checksums and `.SRCINFO` must be updated before pushing to the AUR.
+Unfortunately, Renovate can’t do that ([yet](https://github.com/renovatebot/renovate/tree/feat/arch-linux-manager)), but GitHub Actions can!
```

Comment:
@JamieMagee Is there an issue we can link to? It feels a bit odd to link to a work in progress branch.

Reply:
Yep! https://github.com/renovatebot/renovate/issues/16923

Reply:
```suggestion
Unfortunately, Renovate can’t do that (yet, see [Renovate issue #16923](https://github.com/renovatebot/renovate/issues/16923)), but GitHub Actions can!
```